.github/workflows/build.yml

```diff
@@ -433,7 +433,6 @@ jobs:
       - install
       - monitoring
       - integration-test
-      - workspace-integration-tests-main
```

Comment:
you still need to wait for the task to end

Reply:
thought it shouldn't, as the workspace-integration-tests-main job doesn't use the runner, or am i missing something?

Reply:
You need to wait for the task to end. without the task here, the workflow finish